requirements/bench.txt

```diff
@@ -15,3 +15,5 @@ pyomo>=5.7
 mixsimulator>=0.2.9.5
 hyperopt>=0.2.5
 IOHexperimenter==0.2.8.7
+koncept>=0.2.2
+tensorflow>=2.3.1
```

Comment:
is tensorflow necessary? I don't see it anywhere. At some point we decided not to include it in bench.txt because it's big, then again, bench.txt is already big and not installed by default anyway

Reply:
It's a dependency of koncept and we need to add this line to make sure that the tensorflow version is recent enough to work (at least on my setup on my devfair).